firestore/src/include/firebase/firestore/settings.h

```diff
@@ -137,13 +139,37 @@ class Settings final {
   void set_ssl_enabled(bool enabled);
 
   /**
+   * Returns a reference to the `LocalCacheSettings` instance
+   * used to configure this SDK.
+   */
+  const LocalCacheSettings& local_cache_settings();
+
+  /**
+   * Configures the SDK with the given `LocalCacheSettings` instance.
+   *
+   * By default, persistence cache is enabled, with a cache size of 100 MB.
+   *
+   * See documentation of `PersistentCacheSettings` to under the default
+   * settings.
+   *
+   * @param cache_settings Settings object to configue this SDK.
+   */
+  void set_local_cache_settings(const LocalCacheSettings& cache_settings);
+
+  /**
+   * NOTE: This method is deprecated in favor of `set_local_cache_settings()`.
```

Comment:
nit: Move this deprecation note to _after_ the "Enables or disables local persistent storage" line. Otherwise, the summary documentation in generated docs and IDE hover-overs will simply read as the deprecation notices. Also in `set_cache_size_bytes()`.